common/app/com/commercetools/sunrise/framework/viewmodels/forms/QueryStringUtils.java

```diff
@@ -50,6 +48,20 @@ private QueryStringUtils() {
     }
 
     /**
+     * Extracts the query string from the request.
+     * @param httpRequest current HTTP request
+     * @param ignoredParams params that should not be included in the link
+     * @return query string from the request
+     */
+    public static Map<String, List<String>> extractQueryString(final Http.Request httpRequest,final Set<String> ignoredParams) {
```

Comment:
We don't have many tests in Sunrise yet, but we actually have one for this class 😅 
Could you write some tests for this method?